packages/tabs/src/vaadin-tabs.js

```diff
@@ -187,15 +189,19 @@ class Tabs extends ElementMixin(ListMixin(ThemableMixin(PolymerElement))) {
     });
   }
 
+  /** @protected */
+  _onResize() {
+    this._updateOverflow();
+  }
+
   /** @private */
   __tabsItemsChanged(items) {
     // Disconnected to unobserve any removed items
-    this.__resizeObserver.disconnect();
-    this.__resizeObserver.observe(this);
+    this.__itemsResizeObserver.disconnect();
 
     // Observe current items
     (items || []).forEach((item) => {
-      this.__resizeObserver.observe(item);
+      this.__itemsResizeObserver.observe(item);
```

Comment:
Could reuse the resize observer from the mixin here. Would require to unobserve the removed items individually. This would not work if we decide to use a single observer for all elements in the mixin.

Reply:
The `.disconnect();` above would have an unwanted side-effect of also unobserving the host.

Reply:
Yes, the disconnect would have to be replaced with unobserving the removed items individually. Anyway probably not worth it to optimize this, feel free to resolve.